apacheconfig/__init__.py

```diff
@@ -7,6 +7,8 @@
 from apacheconfig.parser import make_parser
```

Comment:
note to self: we should eventually sort imports alphabetically throughout the code base

Reply:
Opened issue #91 . I'll go through and sort all the imports after this one lands.